central/securedclustercertgen/certificates_test.go

```diff
@@ -115,11 +162,11 @@ func (s *localScannerSuite) TestCertificateGeneration() {
 	}
 }
 
-func (s *localScannerSuite) TestServiceIssueLocalScannerCerts() {
+func (s *securedClusterCertgenSuite) TestServiceIssueLocalScannerCerts() {
```

Comment:
Not sure, but have you considered merging this with `TestServiceIssueSecuredClusterCerts`? The set of service types could parameterize the test cases.

Reply:
They can be merged, but the resulting test would be harder to read imo, so I'd rather have this duplication here.